akka-remote/src/main/scala/akka/remote/artery/BufferPool.scala

```diff
@@ -73,19 +73,18 @@ private[remote] object EnvelopeBuffer {
 
   val VersionOffset = 0 // Byte
   val FlagsOffset = 1 // Byte
-  // 2 bytes free // TODO re-align values to not have this empty space
-  val UidOffset = 4 // Long
-  val SerializerOffset = 12 // Int
+  val UidOffset = 2 // Long
+  val SerializerOffset = 10 // Int
 
-  val SenderActorRefTagOffset = 16 // Int
-  val RecipientActorRefTagOffset = 20 // Int
-  val ClassManifestTagOffset = 24 // Int
-  val ActorRefCompressionTableVersionTagOffset = 28 // Int // TODO handle roll-over and move to Short
-  val ClassManifestCompressionTableVersionTagOffset = 32 // Int // TODO handle roll-over and move to Short
+  val SenderActorRefTagOffset = 14 // Int
+  val RecipientActorRefTagOffset = 18 // Int
+  val ClassManifestTagOffset = 22 // Int
+  val ActorRefCompressionTableVersionOffset = 26 // Byte
+  val ClassManifestCompressionTableVersionOffset = 27 // Byte
```

Comment:
shall we move these up, before all the Tag thingies? (these are ordinary fields)